src/test/resources/log4j2-test.properties

```diff
@@ -0,0 +1,19 @@
+status = error
```

Comment:
Does this file get picked up automatically by log4j2?

Reply:
The upgrade instructions on this file don't include any instructions on loading-so I've just assumed it's picked up automatically somewhere in the building or logging process?

Reply:
Do the upgrade instructions state that this file is needed?

Please could you get confirmation of whether this is necessary from the Core Platform folks.

Reply:
The instructions say the file is needed "to have structured logging for tests."  I'm happy to ask about it, but I can't find a core-platform channel in slack-is it private?  Is there a person I could contact?

Reply:
> The instructions say the file is needed "to have structured logging for tests." 

That suggests it is automatically picked up somehow. It would be interesting to know if that is actually the case.

> I'm happy to ask about it, but I can't find a core-platform channel in slack-is it private? Is there a person I could contact?

The place to ask is the raml-module-builder slack channel.

Reply:
The answer I got was a bit cryptic, but I think I'm being told the file isn't required.  I removed it from the build.

Reply:
Thanks @K-Felk for digging into this

Reply:
@K-Felk Have you pushed the changes where this file was removed?

Reply:
I was having some git problems-it was showing the file was deleted locally and there were no changes to push, but it was still showing here.  Looks like the local branch somehow got out-of-sync with the remote, I had to delete and re-clone.  Looks like it's going through now.